### ISSUE TITLE
[BUG] Violentmonkey refuse to execute my script on google on opera browser

### ISSUE THREAD

**Issue by federicorosso1993** [violentmonkey/violentmonkey#2121]
### Sequence of actions: 
1. I got a personal simple script that I use to change the background on google.com (only on the search page)
2. Violentmokey was working before but now it say that it will not execute script on opera search engine website on opera browser for no reason.

#### Environment:
- OS: Windows
- Browser: Opera 110.0.5130.82
- Violentmonkey Version: 2.19.0

**Comment by federicorosso1993** [comment]
Also I just tried my script and it's still working on Tampermonkey so the bug is on violentmonkey for sure.

**Comment by federicorosso1993** [comment]
I talked too soon... both work if I manually deactivate/activate "Allow access to search results" but only until I restart the browser.

**Comment by tophf** [comment]
Indeed, in Opera you need to manually allow the extension to run on the default search page in `chrome://extensions`, but it's not a problem in Violentmonkey. Unfortunately there's nothing in the description that I can use to reproduce the problem, so I can't fix it. There's another way to find the problem: you can try finding the last working version using https://github.com/violentmonkey/violentmonkey/releases/

**Comment by federicorosso1993** [comment]
https://i.imgur.com/3RX6aA3.png
it's already allowed.
Do you think this is a problem with Opera?

**Comment by tophf** [comment]
I have no idea.

**Comment by tophf** [comment]
Try debugging your script in devtools.

**Comment by federicorosso1993** [comment]
it's not a problem with the script, when I allow it to run on search engine after deactivating it, it will work until I restart Opera.
The strange thing is that violentmonkey say something like "this cannot work on search engine on opera" that's why I opened this bug report
I will try to see if any of the previous version of violentmonkey works.

**Comment by tophf** [comment]
> this cannot work on search engine on opera

It means that the option in `chrome://extensions` didn't work, so apparently it's a bug in Opera.

**Comment by federicorosso1993** [comment]
> > this cannot work on search engine on opera
> 
> It means that the option in `chrome://extensions` didn't work, so apparently it's a bug in Opera.

ok I think you are right. I got back to version 2.15.0 and it still does not work... but I'm sure that version was working fine. The only difference is that it will not show the message that I was talking about.

**Comment by tophf** [comment]
Try updating Opera to 111.0.5168.25, it works for me.

**Comment by federicorosso1993** [comment]
> Try updating Opera to 111.0.5168.25, it works for me.

I think I'm already on that version it had updated while I was writing this. But this still not work....
I notice now that I got the same problem with another extension called UI Changer for Reddit (when I go on reddit from google it will not work right now).
So I'm now sure it is a problem with opera

**Comment by tophf** [comment]
Open `chrome://policy` and see if it has `ExtensionSettings` with `runtime_blocked_hosts` inside.

**Comment by federicorosso1993** [comment]
> chrome://policy

this will not work on my browser at all.
opera will turn it into opera://policy/ but still return an empty page.

**Comment by tophf** [comment]
Ah, Opera doesn't support these policies, so it's not what causes the problem anyway.
